### DISCUSSION TITLE
Proposal: Add Ian to @bokeh/core

### DISCUSSION THREAD

**Discussion by bryevdv** [bokeh/bokeh#12015] [Governance]
@ianthomas23 has made [extensive contributions to webgl and mathtext](https://github.com/bokeh/bokeh/pulls?q=is%3Apr+author%3Aianthomas23+is%3Aclosed) in Bokeh, and has expressed intention to continue contributing in these and other areas. I propose to add him to @bokeh/core

cc @bokeh/core please add 👍 / 👎 on this issue. (Votes will be recorded in one week.)

**Comment by bryevdv** [comment]
<img width="283" alt="Screen Shot 2022-03-02 at 08 04 03" src="https://user-images.githubusercontent.com/1078448/156399842-2864fb53-149c-4b19-93da-fa63a27228d1.png">

Proposal accepted!

**Reply by ianthomas23** [reply]
Thanks @bokeh/core, it is good to be on board!
